operator/v1/types_etcd.go

```diff
@@ -43,6 +43,18 @@ type EtcdSpec struct {
 	// +openshift:enable:FeatureGate=HardwareSpeed
 	// +optional
 	HardwareSpeed ControlPlaneHardwareSpeed `json:"controlPlaneHardwareSpeed"`
+
+	// backendQuotaGiB sets the etcd backend storage size limit in gibibytes.
+	// The value should be an integer not less than 8 and not more than 32.
+	// When not specified, the default value is 8.
+	// +kubebuilder:default:=8
+	// +kubebuilder:validation:Minimum=8
+	// +kubebuilder:validation:Maximum=32
+	// +kubebuilder:validation:XValidation:rule="self>=oldSelf",message="etcd backendQuotaGiB may not be decreased"
+	// +openshift:enable:FeatureGate=EtcdBackendQuota
+	// +default:=8
```

Comment:
Try this, it should change the default value in the openapi.json file, syntax must be wrong here
```suggestion
	// +default=8
```

Reply:
@JoelSpeed 

```
Error running codegen: error running generator openapi: could not generate openapi definitions: error executing openapi generator: Failed executing generator: some packages had errors:
failed to generate default in github.com/openshift/api/operator/v1.EtcdSpec: BackendQuotaGiB: invalid default value (8) for non-pointer/non-omitempty. If specified, must be: 0
make: *** [update-openapi] Error 255
```

Reply:
Please add an `omitempty` tag, you have an optional field with a default, it should be `omitempty`

Reply:
done 👍🏽